baseledger/x/bridge/types/params.go

```diff
@@ -26,7 +26,7 @@ func NewParams() Params {
 func DefaultParams() Params {
 	return Params{
 		WorktokenEurPrice:       "0.1",
-		BaseledgerFaucetAddress: "baseledger1xgs5tamqre7rkz5q7d5fegjsdwufxxvt36w0a0",
+		BaseledgerFaucetAddress: "baseledger1p8x9ud2m75dmufevmrym3uak0hgcrw58h6n872",
```

Comment:
You used another address compared to our last test session?

Reply:
because i don't have mnemonic for this one that was set here, and i need mnemonic for testnet script 
not sure how we can handle this, will think about it maybe through env variable if it's accessible here...